pkg/corerp/api/README.md

```diff
@@ -46,7 +46,8 @@ The following configuration generates track2 go models and client.
 
 ```yaml $(tag) != ''
 version: 3.*.*
-use: "@autorest/go@4.0.0-preview.55"
+use: "@autorest/go@4.0.0-preview.69"
+module: "github.com/radius-project/radius/pkg/corerp/api/v20231001preview"
```

Comment:
was `module` made a required property in the new version?

Reply:
yes, for azure-arm=true, both module and module-version are now required.
sample ref: [link](https://github.com/Azure/autorest.go/blob/7afad1d855757335be8b6429990a71e1210f7a47/packages/codegen.go/src/constants.ts#L23)